docker/docker-compose.dev.yml

```diff
@@ -12,6 +12,9 @@ services:
     volumes:
       - ..:/srv/app
     command: "flask run --host=0.0.0.0"
```

Comment:
the default `SIGTERM` works correctly here, stopping the Flask local web server in ~1s